### PR TITLE
[IMP] accounting: updated chart of accounts page

### PR DESCRIPTION
**Pull request by toaa-odoo**
Updated the chart of accounts page. Forward to master.

16.3 -> Change "ellipsis" menu button
17 -> Change "Setup" to "View"

taskid-3609365

### CODE REVIEW

File: content/applications/finance/accounting/get_started/chart_of_accounts.rst [hunk 1]
```diff
@@ -104,61 +93,58 @@ right type among the following list:
 Assets, deferred expenses, and deferred revenues automation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Some **account types** display a new field **to automate** the creation of :ref:`assets
-<assets-automation>` entries, :ref:`deferred expenses <deferred-expenses-automation>` entries,
-and :ref:`deferred revenues <deferred-revenues-automation>` entries.
+Some **account types** can **automate** the creation of :ref:`assets <assets-automation>` entries,
+:ref:`deferred expenses <deferred-expenses-automation>` entries, and
+:ref:`deferred revenues <deferred-revenues-automation>` entries. To **automate** entries, click on
```
Comment (zulu-odoo) [inline content/applications/finance/accounting/get_started/chart_of_accounts.rst:98]:
```suggestion
:ref:`deferred revenues <deferred-revenues-automation>` entries. To **automate** entries, click
```

Removed 'click ON'

File: content/applications/finance/accounting/get_started/chart_of_accounts.rst [hunk 2]
```diff
@@ -104,61 +93,58 @@ right type among the following list:
...
 
 #. :guilabel:`No`: this is the default value. Nothing happens.
 #. :guilabel:`Create in draft`: whenever a transaction is posted on the account, a draft entry is
    created, but not validated. You must first fill out the corresponding form.
-#. :guilabel:`Create and validate`: you must also select a **model**. Whenever a transaction is
-   posted on the account, an entry is created and immediately validated.
+#. :guilabel:`Create and validate`: you must also select a :guilabel:`Deferred Expense Model`.
+   Whenever a transaction is posted on the account, an entry is created and immediately validated.
 
 Default taxes
```
Comment (zulu-odoo) [inline content/applications/finance/accounting/get_started/chart_of_accounts.rst:109]: For idiots like me, please specify where to go for this (click 'Setup')

File: content/applications/finance/accounting/get_started/chart_of_accounts.rst [hunk 3]
```diff
@@ -104,61 +93,58 @@ right type among the following list:
...
-Select a **default tax** that will be applied when this account is chosen for a product sale or
-purchase.
+Select a **default tax** to be applied when this account is chosen for a product sale or purchase.
 
 Tags
 ----
 
-Some accounting reports require **tags** to be set on the relevant accounts. By default, you can
-choose among the tags that are used by the *Cash Flow Statement*.
+Some accounting reports require **tags** to be set on the relevant accounts. To add a tag, click on
```
Comment (zulu-odoo) [inline content/applications/finance/accounting/get_started/chart_of_accounts.rst:117]:
```suggestion
Some accounting reports require **tags** to be set on the relevant accounts. To add a tag, click
```

Removed 'on'

File: content/applications/finance/accounting/get_started/chart_of_accounts.rst [hunk 4]
```diff
@@ -104,61 +93,58 @@ right type among the following list:
...
+the :guilabel:`Tags` field and select an existing tag, or :guilabel:`Create` a new one.
 
 Account groups
 --------------
 
 **Account groups** are useful to list multiple accounts as *sub-accounts* of a bigger account and
 thus consolidate reports such as the **Trial Balance**. By default, groups are handled automatically
 based on the code of the group. For example, a new account `131200` is going to be part of the group
-`131000`.
+`131000`. You can attribute a specific group in the :guilabel:`Group` field.
```
Comment (zulu-odoo) [inline content/applications/finance/accounting/get_started/chart_of_accounts.rst:126]:
```suggestion
`131000`. You can attribute a specific group to an account in the :guilabel:`Group` field.
```

I would add "to an account" for clarity, but up to you

File: content/applications/finance/accounting/get_started/chart_of_accounts.rst [hunk 5]
```diff
@@ -104,61 +93,58 @@ right type among the following list:
...
 
 Create account groups manually
-------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. note::
    Regular users should not need to create account groups manually. The following section is only
    intended for rare and advanced use cases.
 
 To create a new account group, :ref:`developer mode <developer-mode>` and head to
```
Comment (zulu-odoo) [inline content/applications/finance/accounting/get_started/chart_of_accounts.rst:135]:
```suggestion
To create a new account group, activate :ref:`developer mode <developer-mode>` and head to
```

I think a verb is missing here

File: content/applications/finance/accounting/get_started/chart_of_accounts.rst [hunk 6]
```diff
@@ -3,48 +3,37 @@ Chart of accounts
...
 
 The **chart of accounts (COA)** is the list of all the accounts used to record financial
-transactions in the general ledger of an organization.
+transactions in the general ledger of an organization. The chart of accounts can be found under
+:menuselection:`Accounting-->Configuration-->Chart of Accounts`.
 
-The accounts are usually listed in the order of appearance in the financial reports. Most of the
-time, they are listed as follows :
+When browsing your chart of accounts, you can notably filter the accounts by :guilabel:`Code`,
+:guilabel:`Account Name`, or :guilabel:`Type`, but other options are available in the dropdown menu
```
Comment (zulu-odoo) [inline content/applications/finance/accounting/get_started/chart_of_accounts.rst:10]: FYI there's dropdown and drop-down spelling in the same doc, I'd choose just one ;)

File: content/applications/finance/accounting/get_started/chart_of_accounts.rst [hunk 7]
```diff
@@ -104,61 +93,58 @@ right type among the following list:
...
-purchase.
+Select a **default tax** to be applied when this account is chosen for a product sale or purchase.
 
 Tags
 ----
 
-Some accounting reports require **tags** to be set on the relevant accounts. By default, you can
-choose among the tags that are used by the *Cash Flow Statement*.
+Some accounting reports require **tags** to be set on the relevant accounts. To add a tag, click on
+the :guilabel:`Tags` field and select an existing tag, or :guilabel:`Create` a new one.
```
Comment (zulu-odoo) [inline content/applications/finance/accounting/get_started/chart_of_accounts.rst:118]:
```suggestion
the :guilabel:`Tags` field and select an existing tag or :guilabel:`Create` a new one.
```

**Review by zulu-odoo — CHANGES_REQUESTED** [review]
Enjoyyy

File: content/applications/finance/accounting/get_started/chart_of_accounts.rst (hunk 2 above)
Comment (toaa-odoo) [reply content/applications/finance/accounting/get_started/chart_of_accounts.rst:109]: It's specified on line 99

**Review by zulu-odoo — APPROVED** [review]
Fabulous PR Mr. Tom!

File: content/applications/finance/accounting/get_started/chart_of_accounts.rst [hunk 8]
```diff
@@ -104,61 +93,61 @@ right type among the following list:
...
 
 .. image:: chart_of_accounts/account-groups.png
-   :align: center
    :alt: Account groups creation.
 
-To display your **Trial Balance** report with your account groups, go to :menuselection:`Accounting
-app-->Reporting-->Trial Balance`, then open the :guilabel:`Options` menu and select
-:guilabel:`Hierarchy and Subtotals`.
+To display your **Trial Balance** report with your account groups, go to
+:menuselection:`Accounting-->Reporting-->Trial Balance`, then open the :guilabel:`Options` menu and
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/get_started/chart_of_accounts.rst:147]:
```suggestion
:menuselection:`Accounting --> Reporting --> Trial Balance`, then open the :guilabel:`Options` menu and
```

File: content/applications/finance/accounting/get_started/chart_of_accounts.rst [hunk 9]
```diff
@@ -171,15 +160,18 @@ For example, an invoice paid with a credit card can be marked as :guilabel:`paid
...
 
 Deprecated
 ----------
 
 It is not possible to delete an account once a transaction has been recorded on it. You can make
-them unusable by using the **Deprecated** feature.
-
-To do so, check the :guilabel:`Deprecated` box in the account's settings, and save.
+them unusable by using the **Deprecated** feature: check the :guilabel:`Deprecated` box in the
+account's settings, and :guilabel:`Save`.
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/get_started/chart_of_accounts.rst:174]:
```suggestion
them unusable by checking the :guilabel:`Deprecated` box in the
account's settings.
```
Suggestion to shorten

**Review by xpl-odoo — APPROVED** [review]
You are missing spaces in the `:menuselection:`
For the rest, I made a few suggestions to shorten some sentences (you don't have to take them into account if you don't feel like ;))

@robodoo delegate+

**Comment by toaa-odoo** [comment]
@robodoo r+

**Pull request closed without merging**